extensions/wikia/DesignSystem/i18n/build.js

```diff
@@ -1,6 +1,6 @@
 var fs = require('fs'),
 	path = require('path'),
-	rootDir = process.cwd() + '/node_modules/design-system-i18n/i18n',
+	rootDir = process.cwd() + '/node_modules/design-system/i18n',
```

Comment:
Apparently translation job was not working after we moved i18n files to design system repo :)

Reply:
😮